share/completions/eselect.fish

```diff
@@ -33,12 +34,19 @@ function __fish_complete_eselect_action_options
     __fish_eselect_cmd $cmdl[2..-1] | sed -n -re $findregexp | __fish_sgrep '^    --' | sed -re $parseregexp
 end
 
+function __fish_complete_eselect_php_actions
+    set -l sedregexp 's/^  .([0-9]+).   ([A-Za-z0-9\.]+).?+/\1\t\2/'
+
+    if test (__fish_print_cmd_args_without_options)[3] = 'set'
+        eselect php list (__fish_print_cmd_args_without_options)[-1] | sed -r $sedregexp
```

Comment:
Elis <notifications@github.com> writes:

> The AUR thingy seems handy for testing this.

Yes, it is. Very, very handy. I've also written a stupid wrapper script
to allow me to build from a named branch, called [mkfish](https://github.com/faho/config/blob/master/fish/.config/fish/functions/mkfish.fish).

> Why the kernel btw, I skipped that and tested it directly in the chroot. Much faster :)

Well, I've wanted to see what the fuss about Gentoo was about for a
while, so I've wanted to go through a full install, with a typical-ish
system (that also means openrc instead of systemd).

I feel like this should also help me see where our completions are
lacking.